src/object_store.cpp

```diff
@@ -110,8 +110,6 @@ ColKey add_column(Group& group, Table& table, Property const& property)
         auto key = table.add_column(to_core_type(property.type), property.name, is_nullable(property.type));
         if (property.requires_index())
             table.add_search_index(key);
-        if (property.is_primary)
```

Comment:
Setting the property as the primary key here doesn't work because the user hasn't had a chance to populate it yet and all of the values will be the same.